changelog/unreleased/pr-14409.toml

```
@@ -0,0 +1,5 @@
+type = "fixed"
+message = "Reduce logging and add metric for [Couldn't remove default stream] warning."
```

Is there are specific reason for the brackets usage instead of quotes?

```suggestion
message = "Reduce logging and add metric for \"Couldn't remove default stream\" warning."
```